Cargo.toml

```diff
@@ -39,6 +39,7 @@ exclude = [
     "kernel/libs/comp-sys/controlled",
     "osdk",
     "target/osdk/base",
+    "target/osdk/base-test",
```

Comment:
Better rename it as `test-base`? Since it's a base crate for testing. And also you may add some comment here in `Cargo.toml` about what they are (base and test base) and why are they excluded (because they are OSDK-generated).